scripts/build_utils.sh

```diff
@@ -655,6 +655,13 @@ setup_pbuilder_for_new_gcc() {
     local version=$1
     shift
 
+    cat >> $hookdir/D05install-gcc-7 <<EOF
+echo "deb [lang=none] http://ports.ubuntu.com/ubuntu-ports $DIST-updates main" >> \
```

Comment:
1) I think this should be http://archive.ubuntu.com/ubuntu/dists/$DIST-updates, no?
2) it ought to go on the end of /etc/apt/sources.list, as it's part of the ubuntu sources

Reply:
(at least for x86_64; it's possible arm64 is different)

Reply:
@dmick fixed and repushed.